sklearn/notebooks/ml_engine_training.ipynb

```diff
@@ -441,12 +385,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "! gsutil ls gs://$BUCKET_NAME/census_*"
+    "! gsutil ls $JOB_DIR"
```

Comment:
Where the job is saved will need to be updated for this switch.

Reply:
The reason I changed it was because the job was not actually saved in gs://$BUCKET_NAME/census_* and the link is wrong. The job is actually saved in $JOB_DIR